nuxt.config.js

```diff
@@ -31,7 +31,9 @@ export default {
   */
   head: {
     title: pkg.name,
-    script: [],
+    script: [
+      { src: 'https://cloud.ibm.com/analytics/build/bluemix-analytics.min.js', async: 'true' }
```

Comment:
I see you're loading this `async` but don't see any `onload` event to make sure the analytics are in place before trying to call them?